modules/i18n/src/translations/en-US/portals/myaccount.ts

```diff
@@ -901,7 +901,7 @@ export const myAccount: MyAccountNS = {
                 descriptions: {
                     hint: "You'll receive a text message containing an one-time verification code"
                 },
-                heading: "SMS Number",
+                heading: "Mobile number",
```

Comment:
Let's update the other languages as well.

Reply:
Missed. Fixed with https://github.com/wso2/identity-apps/pull/4882/commits/ed6c6fcfe6a6e74a9296996201346826a2e5f7e7